pom.xml

```diff
@@ -101,6 +101,8 @@ flexible messaging model and an intuitive client API.</description>
     <module>tests</module>
     <module>pulsar-log4j2-appender</module>
     <module>protobuf-shaded</module>
+    <!-- jclouds shaded for gson conflict: https://issues.apache.org/jira/browse/JCLOUDS-1166 -->
```

Comment:
we're shading so much crap. there must be a better way.

Reply:
Thanks, @ivankelly, Opened issue #2164 for this. Once it is done we could remove this shading.